pyproject.toml

```
@@ -34,18 +34,26 @@ fawltydeps = "fawltydeps.main:main"
 [tool.poetry.dependencies]
 # These are the main dependencies for fawltydeps at runtime.
 # Do not add anything here that is only needed by CI/tests/linters/developers
-python = "^3.7.2"
-importlib_metadata = "=6.6.0"
+python = ">=3.7.2"
+importlib_metadata = [
+    # v6.8.0 drops support for Python v3.7:
+    {version = ">=6.6.0", python = ">=3.8"},
+    {version = ">=6.6.0,<6.8.0", python = "<3.8"},
```

It would be safer to take a caret requirement here. Major version change may affect our usage.
```suggestion
    {version = "^6.6.0", python = ">=3.8"},
    {version = ">=6.6.0,<6.8.0", python = "<3.8"},
```